command/policy_write.go

```diff
@@ -91,9 +91,14 @@ func (c *PolicyWriteCommand) Run(args []string) int {
 	}
 
 	// Policies are normalized to lowercase
-	name := strings.TrimSpace(strings.ToLower(args[0]))
+	name := args[0]
+	formattedName := strings.TrimSpace(strings.ToLower(name))
 	path := strings.TrimSpace(args[1])
 
+	if name != formattedName {
+		c.UI.Warn(fmt.Sprintf("Policy name was converted to %s", formattedName))
+	}
+
```

Comment:
Does it make sense to move this down after the client call? If we're going to error anyway no sense warning.

Reply:
Good point, would be weird to see the warning before that.